### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by aienuo** [ueberdosis/tiptap#2801] [General Questions]
Hello, everyone. Can't this project use < script > to import JS? I hope someone can help me

**Comment by aienuo** [comment]
Does not support CDN introduction ？

**Comment by codemzy** [comment]
It can - but there are some known issues, so I don't think it's necessarily recommended at the moment (not 100% sure on that). You need to use `<script type="module">` and then import the packages..

```html
  <script type="module">
    import { Editor } from 'https://cdn.skypack.dev/@tiptap/core?min'
```

Here's the docs https://tiptap.dev/installation/cdn

**Comment by aienuo** [comment]
![image](https://user-images.githubusercontent.com/47814002/169656856-d21ec40b-baec-48ad-bc87-95d66e7795bd.png)


There seems to be a problem
